### ISSUE TITLE
Hyperparameter optimization - relationship with hyperparameter importance?

### ISSUE THREAD

**Issue by jeekoong** [optuna/optuna#3879]
### Don't use GitHub Issues to ask support questions.

I am using Optuna for hyperparameter optimization of neural network.
There are 5 parameters:
  dNum, un, fd : layers sizes of model / suggest_int 1 ~ 50 
  lr : learning rate / suggest_discrete_uniform 1e-7 ~ 1e-5
  dc : decay / suggest_discrete_uniform 1e-7 ~ 1e-4

I use Cma-Es sampler instead of TPE, which does not consider relationship between parameters.
And I use wandb for callback.

After several times of retrieving results,
I found that there is some kind of relationship between importance and  optimization.

Example 1
![image](https://user-images.githubusercontent.com/68321185/184090692-258690ff-715d-4f42-8a2e-577b57b4a6b3.png)
![image](https://user-images.githubusercontent.com/68321185/184090722-67dfbd15-92a2-413c-bfdd-2c9cc036208a.png)
Example 2
![image](https://user-images.githubusercontent.com/68321185/184091009-a8c97716-d916-42cc-b3fa-7e0e89a6d76f.png)
![111](https://user-images.githubusercontent.com/68321185/184091193-b2c759c8-5e6b-4a80-9916-8377d5ada1e2.png)

As shown in Example 1 and 2,
all the parameters, except the one that has largest importance, tend to converge to the median of suggested range.
Even with similar importance results, this happens. I think this can cause issues.

I would like to know:
  1. Why this happens?
  2. If it is normal for this tendency to happen?
  3. If I can change this by code modification

Thank you for reading my question.

**Comment by nzw0301** [comment]
As the issue template says 

> Don't use GitHub Issues to ask support questions.

Please use the [GitHub discussions](https://github.com/optuna/optuna/discussions) instead of posting the issue to github issue.
